### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by shaksham08**
Added homepage for Ecom

image
image

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: var(--white);
+  height: 9rem;
+  padding: 0 2.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```
Comment (t007rushi) [inline styles/main.css]: this can be added to utility

File: styles/main.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
+main {
+  padding: 0rem 5rem;
```
Comment (t007rushi) [inline styles/main.css]: only 0 is inuff

File: styles/main.css [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+.nav {
+  height: 10vh;
+  box-shadow: 0px 4px 11px -8px var(--black-primary);
+  -webkit-box-shadow: 0px 4px 11px -8px var(--black-primary);
+  -moz-box-shadow: 0px 4px 11px -8px var(--black-primary);
+}
+
+.nav__link i {
+  margin-bottom: 0.5rem;
+  margin-left: 0.5rem;
```
Comment (t007rushi) [inline styles/main.css]: below 10px rem would be preferred as shadow you used or make change to above for consistency just suggestion

**Review by ozairkhan2810 — APPROVED** [review]
everything looks code

**Review by t007rushi — APPROVED** [review]
just some changes needed

**Pull request merged**